### PR TITLE
flashlist improvements

### PR DESCRIPTION
**Pull request by injas-bigbinary**
@sangameshsomawar _a Please check

### CODE REVIEW

File: storybook/stories/FlashList.stories.js [hunk 1]
```diff
@@ -74,7 +84,12 @@ export const FlashList = () => {
         </Container>
       )}
       estimatedItemSize={53}
-      ListFooterComponent={<Typography>Loading</Typography>}
+      isLoading={isLoading}
+      SkeltonComponent={
```
Comment (sangameshsomawar) [inline storybook/stories/FlashList.stories.js:88]: I think the spell is wrong. It should be `skeleton` right?

**Review by sangameshsomawar — APPROVED** [review]
@injas-bigbinary LGTM posted a minor comment, Please check and merge.

This `isLoading` flag can be only used for initial loading. I think we should support a few new props like`isFetching`, `onRefresh` with our hook https://github.com/bigbinary/neeto-invoice-rn/blob/master/src/hooks/useRefreshByUser.js We can show normal loader with isFetching.

what do you think? if agree please create a new issue and work on that.

**Comment by injas-bigbinary** [comment]
@sangameshsomawar Added refresh hook. Please check

**Review by sangameshsomawar — APPROVED** [review]
@injas-bigbinary LGTM

**Pull request merged**